### DISCUSSION TITLE
Git problem

### DISCUSSION THREAD

**Discussion by jay-robert14** [community/community#53290] [Programming Help]
### Body

Hello, I installed git and I am getting this error when I want to do something: git: fatal unable to auto-detect email address
what should I do?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by matej-kaska** [comment]
This error is everywhere on Google, but here you go.

git config --global user.name "firstName lastName"
git config --global user.email "email@example.com"

**Reply by queenofcorgis** [reply]
Hey thanks for helping to solve this @matej-kaska 👍🏼 And thanks for marking the answer @jay-robert14 !
